tests/core/pyspec/eth2spec/test/helpers/rewards.py

```diff
@@ -3,9 +3,16 @@
 
```

Comment:
looks like just diffs from refactor in this file...

Reply:
yes!